### ISSUE TITLE
Release 3.0.4?

### ISSUE THREAD

**Issue by GLoganDR** [vayser/react-js-pagination#152]
I hope this isn't a bother, but is there any possible way someone could trigger a release for 3.0.4? The tar package being at 2.x in this one is causing some high security alerts for our project. I appreciate your work, everyone!

**Comment by LFabre** [comment]
Yes, a new release would be very helpful. It seems the issue was already fixed, we only need a new version to be published.
It's triggering a lot of Snyk Security issues given the version of the `tar` package.

**Comment by jldust** [comment]
I would love to see a new tagged release because I'm also running into the vulnerability issue for the `tar` package.
